content/200-concepts/100-components/01-prisma-schema/09-indexes.mdx

```diff
@@ -0,0 +1,249 @@
+---
+title: 'Indexes'
+metaDescription: 'How to configure index functionality and add full text indexes'
+hidePage: false
+tocDepth: 2
+preview: true
+---
+
+<TopBlock>
+
+Prisma allows configuration of database indexes, unique constraints and primary key constraints through the `extendedIndexes` preview feature in version `3.5.0` and later.This provides extended syntax options in the Prisma schema language for configuring the length of indexes (MySQL only), the sort order and the index type (PostgreSQL only).
+
+Version `3.6.0` also introduces support for introspection and migration of full text indexes in MySQL and MongoDB through a new `@@fulltext` attribute, available through the `fullTextIndex` preview feature.
+
+<Admonition type="warning">
+
+These changes to index configuration and full text indexes might be **breaking changes** if you have a database that is already making use of these features, so they need to be explicitly enabled via preview features. See [Upgrading from previous versions](#upgrading-from-previous-versions) for details.
```

Comment:
The target of this link now exists twice on this page. It will link to the first one probably.

Reply:
Thanks for catching this, that's sort of annoying as it's going to look clunky linking to both separately. Or could remove the link completely here. Any preferences on what to do with it?

(It does indeed link to the first one)

Reply:
Removed it for now.